OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/CourseOutlineAdapter.java

```diff
@@ -341,28 +341,32 @@ private void getRowViewForLeaf(ViewHolder viewHolder, final SectionRow row) {
                 updateUIForVideo(viewHolder, videoData, videoBlockModel);
             } else if (videoBlockModel.getData().encodedVideos.youtube != null) {
                 final boolean isYoutubePlayerEnabled = config.getYoutubePlayerConfig().isYoutubePlayerEnabled();
-                viewHolder.rowType.setIcon(isYoutubePlayerEnabled ? FontAwesomeIcons.fa_youtube_play : FontAwesomeIcons.fa_laptop);
-                viewHolder.rowType.setIconColorResource(isYoutubePlayerEnabled ? R.color.primaryBaseColor : R.color.primaryXLightColor);
+                viewHolder.rowTitle.setCompoundDrawablesRelativeWithIntrinsicBounds(null, null,
+                        UiUtil.getFontAwesomeDrawable(context, isYoutubePlayerEnabled ? FontAwesomeIcons.fa_youtube_play : FontAwesomeIcons.fa_laptop,
+                                R.dimen.small_icon_size, isYoutubePlayerEnabled ? R.color.neutralXDark : R.color.primaryXLightColor), null);
```

Comment:
we have to show only one color from now.

Reply:
Done